index.html

```diff
@@ -179,5 +179,43 @@ <h2 class="title-2">ROAD TRIP PLANNER</h2>
             </section>
         </section>
     </div>
+    <footer class="footer">
+        <section class="contact-1">
+                <h1 class="title-1">
+                    CONTACT US
+                </h1>
```

Comment:
- [ ] Please change this `h1` tag to any other heading tag `t2`-`t6`. Your web page should contain only one `h1` tag for the main title of the page and then other heading tags depending on the hierarchy of the headings.